qa/L0_model_config/autofill_noplatform_success/python/conflicting_scheduler_ensemble/conflicting_scheduler_ensemble/expected

```diff
@@ -0,0 +1,45 @@
+name: "conflicting_scheduler_ensemble"
```

Comment:
I think we must expect an error here. Model trying to set scheduling_choice of 'dynamic_batching' to a model with 'ensemble_scheduling'

Reply:
The ensemble backend [doesn't call any autocomplete](https://github.com/triton-inference-server/core/blob/4092b56124ee3dece0eb9bce439b700e6d30ff45/src/ensemble_model.cc#L47) functions in the backend and so while the model config has the `ensemble_scheduling` set, the python backend won't call the autocomplete function of the ensemble model.  This test is to protect against any future changes in the ensemble backend where it may start to perform autocomplete functions.

Reply:
That makes sense! I was coming up with the conditions that we check against.